src/main/java/mate/academy/hibernate/relations/dao/impl/CountryDaoImpl.java

```diff
@@ -12,11 +15,33 @@ public CountryDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Country add(Country country) {
-        return null;
+        Session session = null;
```

Comment:
Consider using try-with-resources for the `Session` to ensure it is always closed, even if an exception occurs. However, since you are managing the transaction manually, ensure that the transaction is committed or rolled back before the session is closed.